### PR TITLE
doc: remove sphinx gallery style thats now handled upstream

### LINKED ISSUE #963: sphinx gallery update, mirror mne-python fix
https://github.com/mne-tools/mne-bids/blob/81bccd63f44099af30846af895d46ed0d112483d/doc/_static/style.css#L11-L15

we might need to mirror this change: https://github.com/mne-tools/mne-python/pull/10364

### PR DESCRIPTION
**Pull request by sappelhoff**
copying Eric's fix from https://github.com/mne-tools/mne-python/pull/10364 to close #963 

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Review by agramfort — APPROVED** [review]
but not sure why CIs complain

**Comment by sappelhoff** [comment]
some memory error apparently (windows only)

> numpy.core._exceptions._ArrayMemoryError: Unable to allocate 4.10 GiB for an array with shape (376, 1462765) and data type float64

**Comment by agramfort** [comment]
I am sure it's the test I added months ago for split files as it produces
huge raw arrays.... crap...

**Comment by hoechenberger** [comment]
I pushed a commit that aims to fix the issue, but I haven't had the time to test it locally. But even if it fails it should provide a starting point.

**Comment by hoechenberger** [comment]
So the issue here is that `write_raw_bids()` doesn't accept a `split_size` parameter, hence it will only start splitting if the input `Raw` file size exceeds what can be fit into a single FIFF output file.

**Comment by hoechenberger** [comment]
I think we opened a can of worms here 😅

File: mne_bids/tests/test_read.py [hunk 1]
```diff
@@ -967,19 +970,58 @@ def test_write_read_fif_split_file(tmp_path):
     raw = mne.io.RawArray(data, raw.info)
     big_fif_fname = pathlib.Path(tmp_dir) / 'test_raw.fif'
     raw.save(big_fif_fname, split_size='10MB')
-    raw = _read_raw_fif(big_fif_fname, verbose=False)
-    write_raw_bids(raw, bids_path, verbose=False)
+
+    # write_raw_bids() doesn't support a `split_size` parameter, and would
+    # normally only start splitting FIFF files exceeding 2GB in size. To
+    # write split files smaller than this, we hack our way around things and
+    # manually write two files with a `split` entity.
```
Comment (hoechenberger) [inline mne_bids/tests/test_read.py:977]: But this means reading the data back by passing the first filename won't automatically read the remaining splits in this test. So this is **very** synthetic.

**Comment by hoechenberger** [comment]
Maybe the cleanest solution would be to simply add a private `_SPLIT_SIZE` value to `write.py`, defaulting to the MNE-Python default, but easily adjustable for our tests.
@larsoner Got an opinion on that?

**Comment by hoechenberger** [comment]
Ok I introduced `write._FIFF_SPLIT_SIZE`, which can be monkey-patched during testing.
Seems to do the trick!

**Review by hoechenberger — APPROVED** [review]

**Comment by hoechenberger** [comment]
@sappelhoff Feel free to merge if happy.

**Comment by sappelhoff** [comment]
Thanks for the fix @hoechenberger!

**Pull request merged**